<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageRenderWindow</class>
 <widget class="QWidget" name="ImageRenderWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>686</width>
    <height>553</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image Display</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,0">
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="ImageRenderWidget" name="render_widget" native="true"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="widget_label">
           <property name="text">
            <string>Click on point to display intensity</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0">
       <item>
        <widget class="QToolButton" name="zoom_button">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="PuppetMasterResources.qrc">
           <normaloff>:/Tools/Resources/magnifier_48.png</normaloff>:/Tools/Resources/magnifier_48.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="shortcut">
          <string notr="true"/>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoRaise">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="reset_button">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="PuppetMasterResources.qrc">
           <normaloff>:/Tools/Resources/resize_48.png</normaloff>:/Tools/Resources/resize_48.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="shortcut">
          <string notr="true"/>
         </property>
         <property name="autoRaise">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="show_overlay_button">
         <property name="text">
          <string>Show Overlay</string>
         </property>
         <property name="icon">
          <iconset resource="PuppetMasterResources.qrc">
           <normaloff>:/Tools/Resources/tag_48.png</normaloff>:/Tools/Resources/tag_48.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="save_button">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="PuppetMasterResources.qrc">
           <normaloff>:/Tools/Resources/save_48.png</normaloff>:/Tools/Resources/save_48.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="contrast_slider">
         <property name="maximum">
          <number>8</number>
         </property>
         <property name="pageStep">
          <number>2</number>
         </property>
         <property name="value">
          <number>8</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksAbove</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QScrollBar" name="image_scroll">
     <property name="maximum">
      <number>0</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageRenderWidget</class>
   <extends>QWidget</extends>
   <header>ImageRenderWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="PuppetMasterResources.qrc"/>
 </resources>
 <connections/>
</ui>
